src/App/GetStarted/get-started/ForDevelopers/index.js

```diff
@@ -12,74 +12,105 @@ const brandTitle = process.env.brandTitle;
 
 const Installation = () => (
     <section>
-        <h2 id="installation">Installation with CDN</h2>
-        <p>To quickly add the {brandTitle} Design Guide to your project, include the snippets below in your project.</p>
-
-        <section>
-            <h3>Include in the header</h3>
-            <p>Copy-paste the following CSS code into <CodeTags type="secondary" code="<head>"/> before all the other stylesheets in order to load our CSS.</p>
-            <ComponentPreview language="html" codeFigure>
-                <link rel="stylesheet" href={`${window.location.origin + basename}styles/${brand}.css`} />
-            </ComponentPreview>
-        </section>
-        <section>
-            <h3>Include at the bottom</h3>
-            <p>Many of our components requires the use of JavaScript in order to function. Place the following <CodeTags type="secondary" code="<Script>"/> tag near the end of your pages, right before the closing <CodeTags type="secondary" code="</body>"/> tag to enable them.</p>
-            <ComponentPreview language="html" codeFigure>
-                <script src={`${window.location.origin + basename}scripts/dg.js`} global="true" autoload="true"></script>
-            </ComponentPreview>
-            <p>The <CodeTags type="secondary" code="global"/> attribute adds the script to the global scope. The <CodeTags type="secondary" code="autoload"/> attribute automatically initiates all components when loaded. Great for plug and play.</p>
-        </section>
         <section>
             <h2 id="installation-npm">Installation with NPM</h2>
             <p>If you want to host the Design Guide files in your own app, run the following command.</p>
             <ComponentPreview language="terminal" codeFigure>
-            npm install @swedbankpay/design-guide
+        npm install @swedbankpay/design-guide
             </ComponentPreview>
             <section>
-                <h3>Usage with NPM</h3>
-                <p>When importing the Design Guide javascript, you can choose to either import the whole thing or only the components you want to use.</p>
+                <h3>JavaScript</h3>
+                <p>Some of the Design System components require javascript (not all of them). To achieve this:
```

Comment:
Do you think it would be nice to add the list of the components?

Reply:
I like this. Now, there are _21_ components using JS, which is a lot to list in here 😬. So I'm wondering:
- will adding things decrease the clarity of he message by filling too much content in one place?
   - should we put the list there?
   - put the list somewhere else and put here a link to it?
   - just show a screenshot of the "use javascript" badge we add beside the titles in the component's pages?
- if we list them it means we need to keep it in sync
   - I guess we could make it dynamic, iterating over all the components and filtering only the ones needing JS 🤔
   
   What do you think?

Reply:
Oh, I didn't thought there were so many components 😆 Well.. I have thought about adding a small badge in the component overview place, so perhaps that's the best approach, and just leave this for now 👍